db/migrate/20160621151100_polymorphic_mentions.rb

```diff
@@ -0,0 +1,12 @@
+class PolymorphicMentions < ActiveRecord::Migration
+  def change
+    remove_index :mentions, %i(post_id)
+    remove_index :mentions, %i(person_id post_id)
+    rename_column :mentions, :post_id, :mentions_container_id
+    add_column :mentions, :mentions_container_type, :string, default: "Post", null: false
+    add_index :mentions, %i(mentions_container_id mentions_container_type), name: "index_mentions_on_mc_id_and_mc_type"
```

Comment:
Works for me when I do
```
git checkout develop
RAILS_ENV=development FIXTURES_PATH=spec/fixtures bundle exec rake db:drop db:create db:schema:load db:fixtures:load
... posting post with mention ...
git checkout 1851-mentions-in-comments.new
bundle exec rake db:migrate
```
Does database content make difference here?
MySQL version is `mysqld  Ver 5.5.49-0ubuntu0.14.04.1 for debian-linux-gnu on x86_64 ((Ubuntu))`.